actor-typed/src/main/scala/org/apache/pekko/actor/typed/Props.scala

```diff
@@ -81,6 +81,13 @@ abstract class Props private[pekko] () extends Product with Serializable {
    */
   def withDispatcherFromConfig(path: String): Props = DispatcherFromConfig(path, this)
 
+  /**
+   * Prepend a selection of the mailbox found at the given Config path to this Props.
+   * The path is relative to the configuration root of the [[ActorSystem]] that looks up the
+   * mailbox.
```

Comment:
can you add `@since 1.1.0` on new API methods?

Reply:
Make sense, fixed.